src/python/grpcio_tests/tests/interop/client.py

```diff
@@ -115,7 +146,7 @@ def get_secure_channel_parameters(args):
 def _create_channel(args):
     target = '{}:{}'.format(args.server_host, args.server_port)
 
-    if args.use_tls or args.use_alts:
+    if args.use_tls or args.use_alts or args.custom_credentials_type is not None:
```

Comment:
If we copy this line to aio's interop client, the compute engine credentials should work automatically.

Reply:
One line change and the asyncio interop client just worked. Nice!